.gitignore

```diff
@@ -162,3 +162,4 @@ bin/
 *-window.json
 simgui-ds.json
 simgui.json
+simgui.json
```

Comment:
Looks like we duplicated this line in the .gitignore

Reply:
fixed